tools/setup-dev.sh

```diff
@@ -1,11 +1,13 @@
 #!/bin/bash
 
 dependencies_opensuse_tumbleweed=(
+  "btrfsprogs"
   "git"
   "python3"
   "python3-rados"
   "python3-kiwi"
   "nodejs-common"
+  "npm"
```

Comment:
I thought `npm` is installed via `nodejs-common`? At least that's what `cnf` states:

```
➜ joao@adamastor tmp git:(wip-gravel-multinode) ✗ cnf npm
                     
Program 'npm' is present in package 'nodejs-common', which is installed on your system.
```
and rightly so
```
➜ joao@adamastor tmp git:(wip-gravel-multinode) ✗ rpm -ql nodejs-common
/usr/bin/node
/usr/bin/npm
/usr/bin/npx
/usr/share/licenses/nodejs-common
/usr/share/licenses/nodejs-common/LICENSE
```

Reply:
@jecluis this is because `--no-recommends` behavior which is enabled in the openstack image in the /etc/zypp/zypp.conf (see for ```solver.onlyRequires = true```)
And your box, same as mine has that line commented out.

Reply:
Just for info, if you install `nodejs-common` with `--no-recommends` it will not install `npm` package as a result `/usr/bin/npm` will as follows:
```
> npm --version
npm-default is unavailable.
```

Reply:
Alright. Good to know. Thanks for clarifying!